dev-libs/tllist/tllist-1.0.5.ebuild

```diff
@@ -0,0 +1,29 @@
+# Copyright 2020-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit meson
+
+if [[ ${PV} != *9999* ]]; then
```

Comment:
Are you sure want to keep+maintain 9999 ebuilds? Just asking in case, esp given keeping them updated can be more of a pain with proxy-maint (but it's fine you want them).

Could remove this conditional 9999 cruft if not.